natgeo/mortar
# AEM-6933: Update issue date render to use issueLink object

https://jira.natgeo.com/browse/AEM-6933

To test:

- Pull AEM-6933
- run gulp serve
- Verify issue date is rendering properly
- Change the issueLink object around and remove it entirely, verify issue date responds appropriately
